cypress/features/regression/confirm.feature

```diff
@@ -95,3 +86,34 @@ Feature: Confirm component
       | error     | iconError    |
       | warning   | iconWarning  |
       | empty     | iconEmpty    |
+
+  @positive
+  Scenario: cancel button type is set to tertiary
```

Comment:
```suggestion
  Scenario: Cancel button type is set to tertiary
```

Reply:
it is not a component but only button name

Reply:
updated